little code bug report: pass []any as any in variadic function

<!-- Please answer these questions before submitting your issue. Thanks! -->

### What version of Vim/Gvim are you using?

<details><summary>Vim/Gvim version</summary><br><pre>
$ vim --version

</pre></details>

### What version of Go are you using (`go version`)?

<pre>
$ go version

</pre>

### What operating system and processor architecture are you using (`go env`)?

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env

</pre></details>


### What version/commit of `govim` are you using?


### What is the output of `:messages` in Vim?


### Do the Vim channel, `govim` or `gopls` log files show anything interesting?

<!-- !! BEWARE THESE LOGFILES CONTAIN FILE CONTENTS !! -->

<!-- The location of the logfiles is given by :GOVIMLogfilePaths in Vim -->


### How did you install `govim`?

<!-- Vim 8 packages, Pathogen, vim-plug, Vundle, other...  -->


### What did you do?

<!-- If possible, provide a recipe for reproducing the error.  -->

i am writing a [linter](https://github.com/alingse/asasalint) to lint that pass []any as any in variadic function

and run github Actions for some top go packages, this package failed

see github actions lint result here https://github.com/alingse/asasalint/runs/7286692753?check_suite_focus=true

```
Error: /home/runner/work/asasalint/asasalint/govim/channel_cmds.go:10:57: pass []any as any to func handleChannelValueAndError func(ch github.com/govim/govim.unscheduledCallback, err error, format string, args ...interface{}) (encoding/json.RawMessage, error)
Error: /home/runner/work/asasalint/asasalint/govim/cmd/govim/batch.go:76:48: pass []any as any to func append func([]interface{}, ...interface{}) []interface{}
...
```

the first report is true, and the second and etc is a wrong report.

but i don't know if it was by design ? (maybe? )

```Go
func (g *govimImpl) handleChannelError(ch unscheduledCallback, err error, format string, args ...interface{}) error {
	_, err = g.handleChannelValueAndError(ch, err, format, args)
	return err
}
-------
func (g *govimImpl) handleChannelValueAndError(ch unscheduledCallback, err error, format string, args ...interface{}) (json.RawMessage, error) {
	if err != nil {
		return nil, err
	}
	args = append([]interface{}{}, args...)
	select {
	case <-g.tomb.Dying():
		panic(ErrShuttingDown)
	case resp := <-ch:
		if resp.errString != "" {
			args = append(args, resp.errString)
			return nil, fmt.Errorf(format, args...)
		}
		return resp.val, nil
	}
}

```

### What did you expect to see?


### What did you see instead?

--- a/channel_cmds.go
+++ b/channel_cmds.go
@@ -7,7 +7,7 @@ import (
 )
 
 func (g *govimImpl) handleChannelError(ch unscheduledCallback, err error, format string, args ...interface{}) error {
-	_, err = g.handleChannelValueAndError(ch, err, format, args)
+	_, err = g.handleChannelValueAndError(ch, err, format, args...)
 	return err
 }
 

